tools/Resources.py

```diff
@@ -0,0 +1,60 @@
+import os
+import requests
+from urllib3 import disable_warnings, exceptions
+from requests.auth import HTTPBasicAuth
+
+
+class Resources:
+
+    def get_resources(self, target, **kwargs):
+        url = "https://" + target + "/suite-api/api/resources"
+        querystring = {
+            'parentId': kwargs['parentid'],
+            'adapterKind': 'VMware',
+            'resourceKind': kwargs['resourcekind'],
+            'pageSize': '50000'
+        }
+        headers = {
+            'Content-Type': "application/json",
+            'Accept': "application/json"
+        }
+        disable_warnings(exceptions.InsecureRequestWarning)
+        resources = list()
+        response = requests.get(url,
+                                auth=HTTPBasicAuth(username=os.environ['USER'], password=os.environ['PASSWORD']),
+                                params=querystring,
+                                verify=False,
+                                headers=headers)
+        for resource in response.json()["resourceList"]:
```

Comment:
check with hasattr() if this key really exists before calling it

Reply:
resolved